journal-of-software.csl

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Journal of Software(Chinese)</title>
+    <title-short>JOS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-software</id>
+    <link href="http://www.zotero.org/styles/journal-of-software" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-with-titles" rel="template"/>
+    <author>
```

Comment:
(@damnation333, I appreciate the help, but I asked these exact same two questions directly above :P)